info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 1    | 1.2.1    | PACKAGE_NAME_INCORRECT_SYMBOLS            | Check: warns if a package name has incorrect symbols like underscore or non-ASCII letters/digits.Exception: underscores that are used for differentiating of keywords in a name.<br>Fix: no fix currently; will be suggested later                              | no  | no                                                                                                                                                                                                                         | Add autofix for at least converting underscore to a dot or replacing itFix the directory where the code is stored.Cover autofix with tests                                                                                    |
+| 1    | 1.2.1    | PACKAGE_NAME_INCORRECT_PATH               | Check: warns if the path for a file does not match with a package name<br>Fix: replacing the incorrect package name with the name constructed from a path to the file.                                                                                           | yes | no                                                                                                                                                                                                                         | Make this check isolated from domain name creation.Recursively update all imports in the project.Fix the directory where the code is stored.Add a test mechanism to test checker.                                                 |
+| 1    | 1.2.1    | INCORRECT_PACKAGE_SEPARATOR               | Check: warns if the underscore is incorrectly used to split the package naming<br>Fix: fixing all nodes in AST and the package name to remove all underscores                                                                                                        | no  | no                                                                                                                                                                                                                         | Recursively update usages of this class in the project                                                                                                                                                                        |
```

Comment:
"Check: warns if the underscore is incorrectly used to split the package naming." This is not clear to me. Please, explain.

Reply:
This is about [package declarations](https://kotlinlang.org/docs/packages.html) in the Kotlin language.

This is correct:

```kotlin
package com.example.someModule
```

&mdash; while this is not:

```kotlin
package com.example.some_module
```

Neither is this:

```kotlin
package com.example.someModule_
```

Reply:
I changed it to read:

Check: warns if the underscore is incorrectly used in package naming to split name parts.